vscode-pets-integration/experiment2/scripts/create_idle_gif.py

```
@@ -0,0 +1,173 @@
+    
+    os.makedirs(output_dir, exist_ok=True)
+    
+    cmd = [
+        'ffmpeg',
+        '-i', video_path,
+        '-vf', f'fps={fps}',
+        os.path.join(output_dir, 'idle_frame_%03d.png'),
+        '-y'
+    ]
+    
+    result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)